website/sidebars.js

```diff
@@ -254,6 +254,17 @@ const sidebarSettings = {
               "docs/cloud/dbt-cloud-ide/lint-format",
             ],
           },
+          {
+            type: "category",
+            label: "dbt Assist",
```

Comment:
FYI I'm thinking of now adding this back to live inside the dbt cloud ide folder. This is to also make way for an addl dev tool that should live under 'dev w dbt cloud'.

So am going to change this on Monday 🙏